rhoas/tests/kafka_test.go

```diff
@@ -17,8 +17,10 @@ import (
 )
 
 const (
-	kafkaID   = "test_kafka"
-	kafkaPath = "rhoas_kafka.test_kafka"
+	kafkaID           = "test_kafka"
+	kafkaPath         = "rhoas_kafka.test_kafka"
+	PlanInput         = "developer.x1"
```

Comment:
prob. would be better to lowercase them as it's just for testing, no exporting needed